.harness/default_input_set.yaml

```diff
@@ -0,0 +1,14 @@
+inputSet:
```

Comment:
Does this have to be in a separate file? Does harness require this or can you have a multi-doc YAML file, e.g.
```yaml
inputSet:
  name: default_input_set
  identifier: default_input_set
  orgIdentifier: AGENTS
  projectIdentifier: airflowproviderdatarobot
  ...

---
pipeline:
  name: lint_check
  identifier: lint_check
  projectIdentifier: airflowproviderdatarobot
  orgIdentifier: AGENTS
  ...
```

Reply:
Or if this is what the Harness UI generates, so be it 🤷‍♂️